tests/e2e/steps/steps_def.go

```diff
@@ -165,6 +165,26 @@ func (step *Step) validateDriverNotInstalled(res Resource, driverName string, cr
 	return checkNoRunningPods(res.CustomResource[crNum-1].Namespace, step.clientSet)
 }
 
+func (step *Step) setNodeLabel(res Resource, label string) error {
+	if label == "control-plane" {
+		setNodeLabel(label, "node-role.kubernetes.io/control-plane", "")
+	} else {
+		return fmt.Errorf("Setting worker node role to %s not supported, feel free to add support", label)
+	}
+
+	return nil
+}
+
+func (step *Step) removeNodeLabel(res Resource, label string) error {
+	if label == "control-plane" {
+		removeNodeLabel(label, "node-role.kubernetes.io/control-plane")
+	} else {
+		return fmt.Errorf("Setting worker node role to %s not supported, feel free to add support", label)
```

Comment:
should this be a different error message from setNodeLabel?

Reply:
fixed